src/Components/Navbar/navbar.js

```diff
@@ -1,9 +1,33 @@
 import React from 'react'
 import './navbar.css'
+import { Link } from 'react-router-dom'
 
 const Navbar = () => {
   return (
-    <div>Navbar</div>
+    <div>
+      <header className='header'>
+      <a className="logo wow-shop">
+        <Link to='/'><i className='fas fa-shopping-basket'></i> Wow shop</Link>
+      </a>
+
+      <nav className="navbar">
+        <Link to='/'><a>home</a></Link>
+        {/* <a href="#speciality">Offer</a> */}
+        <Link to='/products'><a>products</a></Link>
+        <Link to='/wishlist'><a>Wishlist</a></Link>
+        <Link to='/cart'><a>Cart</a></Link>
+        {/* <a href="#home">Login</a> */}
```

Comment:
If the commented out code is not needed in the future, you can remove it.

Reply:
Okayy!!